forgevtt-module.js

```diff
@@ -2081,22 +2187,24 @@ class ForgeVTT_FilePicker extends FilePicker {
                     <button type="button" name="forgevtt-new-folder" style="line-height: 1rem;">
                         <i class="fas fa-folder-plus"></i>New Folder
                     </button>
-                </div>`)
+                </div>`);
                 upload.hide();
-                upload.after(uploadDiv)
+                upload.after(uploadDiv);
                 uploadDiv.append(upload);
-                uploadDiv.find('button[name="forgevtt-upload"]').click((_ev) => upload.click());
-                uploadDiv.find('button[name="forgevtt-new-folder"]').click((_ev) => this._onNewFolder());
+                uploadDiv.find('button[name="forgevtt-upload"]').on("click", (_ev) => upload.click());
+                uploadDiv.find('button[name="forgevtt-new-folder"]').on("click", (_ev) => this._onNewFolder());
             }
         }
 
+        const select = html.find('select[name="bucket"]');
+        select.off("change").val(this.source.bucket).on("change", this._onChangeBucket.bind(this));
```

Comment:
And here, we remove the "change" event listener rather than just override it, because v12 replaced `_onBucketChange` with a private `#onBucketChange`.